### DISCUSSION TITLE
I can't play single draft even if I already selected all pick and then single draft mode. My game was abandoned because of an interner issue. Please help

### DISCUSSION THREAD

**Discussion by Rodeldota2** [ValveSoftware/Dota2-Gameplay#7926] [General]
I can't play single draft even if I already selected all pick and then single draft mode. My game was abandoned because of an interner issue. Please help

**Comment by run3unblockedaz** [comment]
This is undeniably an excellent piece of writing. We are indebted to you for taking the time to explain all of this in such  detail for all of us. Your website address is:<a href="https://play-snake.co/"> **play snake**</a> It served as an excellent guidance!

**Comment by commentator24** [comment]
#3644
